stories/Button.stories.js

```diff
@@ -60,35 +60,17 @@ stories.add('Link', () =>
   <Button waves='light' node='a' href='http://react-materialize.github.io/'>Open Me In New Tab</Button>
 );
 
-stories.add('Top FAB', () =>
-  <Button floating fab='top' icon='edit' className='red' large>
+stories.add('Vertical FAB', () =>
+  <Button floating fab='vertical' icon='edit' className='red' large>
```

Comment:
That PR will directly affect this one, since stories will change. We can close this one and I can create stories later or in that PR. What you think is the best approach?

:smile:

Reply:
If this is a easy fix, I would keep the changes here. It would be easier to find in the future. Can this be done here?

Reply:
Up to you @onluiz

Reply:
I fix stories in here when the other PR is merged then.

Reply:
@alextrastero, now I think we did it 😄 could you give a look at my last commit?

And thanks for your patience 😆